web/src/narrow.js

```diff
@@ -792,6 +793,8 @@ export function update_selection(opts) {
         message_lists.current.view.set_message_offset(select_offset);
     }
     unread_ops.process_visible();
+    narrow_history.save_narrow_state();
+    narrow_history.save_narrow_state.flush();
```

Comment:
This kinda feels like an abstraction violation; can this be just `narrow_history.save_and_flush_narrow_state` or something like that, and the `.flush` internals of the debouncing not be exposed here?

Reply:
Yeah great point! I just created a `save_narrow_state_and_flush` function.